cmd/minishift/cmd/start.go

```diff
@@ -138,6 +145,7 @@ func runStart(cmd *cobra.Command, args []string) {
 		RegistryMirror:   registryMirror,
 		HostOnlyCIDR:     viper.GetString(hostOnlyCIDR),
 		OpenShiftVersion: viper.GetString(openshiftVersion),
+		ProxyInfo:        proxyUrl,
```

Comment:
-1, the way to do this is via _cluster.RegistrationParameters_. It is not the best either, but that's how it was indented to be done.